stable/mysql/values.yaml

```diff
@@ -133,6 +133,9 @@ resources:
   requests:
     memory: 256Mi
     cpu: 100m
+  limits:
+    memory: 512Mi
+    cpu: 200m
```

Comment:
Limits, like these, are usually commented out and documented. Someone may want a different value when running in a developer environment vs running in production. What is the reason for providing the values?